### PR TITLE
docs: add `http.version` to lbws manifest

### PR DESCRIPTION
**Pull request by efekarakus**
Also parse the Dockerfile for port `50051` that's commonly used for gRPC and set in the manifest file.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/manifest/lb_web_svc.go [hunk 1]
```diff
@@ -23,6 +23,10 @@ const (
 	DefaultHealthCheckGracePeriod = 60
 )
 
+const (
+	GRPCProtocol = "gRPC" // GRPCProtocol is the HTTP protocol version for gRPC.
```
Comment (iamhopaul123) [inline internal/pkg/manifest/lb_web_svc.go:27]: We should probably want to be consistent with
```go
	httpProtocolVersions = []string{"GRPC", "HTTP1", "HTTP2"}
```

**Review by iamhopaul123 — APPROVED** [review]
LGTM just a very tiny little nit

File: internal/pkg/manifest/lb_web_svc.go (hunk 1 above)
Comment (efekarakus) [reply internal/pkg/manifest/lb_web_svc.go:27]: Those values are the ones that are required by CFN, whereas the typical spelling is ["gRPC"](https://grpc.io/). We just ignore the case and move everything to upper case later when transforming the user input, does that make sense?

Comment (iamhopaul123) [reply internal/pkg/manifest/lb_web_svc.go:27]: Yeah that makes sense and also why i said this is just very tiny little nit. it's just i think we'd better to be consistent. But yeah if they are all upper case sure that sounds good to me. Maybe the var name should be `httpProtocolVersionsUpperCase`? You decide boss.

File: internal/pkg/initialize/workload.go [hunk 2]
```diff
@@ -298,16 +303,23 @@ func (w *WorkloadInitializer) newServiceManifest(i *ServiceProps) (encoding.Bina
 }
 
 func (w *WorkloadInitializer) newLoadBalancedWebServiceManifest(i *ServiceProps) (*manifest.LoadBalancedWebService, error) {
+	var httpVersion string
+	if i.Port == commonGRPCPort {
+		log.Infof("Detected port %s, setting HTTP protocol version to %s in the manifest.\n",
+			color.HighlightUserInput(strconv.Itoa(int(i.Port))), color.HighlightCode(manifest.GRPCProtocol))
+		httpVersion = manifest.GRPCProtocol
```
Comment (Lou1415926) [inline internal/pkg/initialize/workload.go:310]: Nice!

**Review by Lou1415926 — APPROVED** [review]

File: internal/pkg/initialize/workload.go (hunk 2 above)
Comment (Lou1415926) [reply internal/pkg/initialize/workload.go:310]: (Adding the label just in case you want to make more changes boss)

**Pull request merged**